sh/prepareData_C2HDM.py

```diff
@@ -21,10 +21,10 @@ def convert(InputFile, OutputFile):
     Lambda2='L2'
     Lambda3='L3'
     Lambda4='L4'
-    re_Lambda5='re_L5'
-    im_Lambda5='im_L5'
+    re_Lambda5='L5r'
+    im_Lambda5='L5i'
     tanbeta='tbeta'
-    re_m12squared='re_m12sq'
+    re_m12squared='m12sqr'
```

Comment:
Why do these need to change?

Reply:
This was in order to fit to the `ScannerS` output convention.

Reply:
Shall I remove this change as it has nothing to do with the aim of the PR?

Reply:
Yes, but you can make a seperate PR for it. Do the other scripts need to be adjusted as well?

Reply:
I reverted the file for this PR. It is possible that other scripts need to be adjusted as well. What about opening an issue for this?

Reply:
yes, please do that. Afterwards you can close this conversation for the merge